lib/sidekiq_unique_jobs/exception.rb

```diff
@@ -0,0 +1,3 @@
+module SidekiqUniqueJobs
+  class RunLockFailedError < StandardError; end
```

Comment:
can we rename this file to `run_lock_failed_error.rb` instead? I prefer it if a class or module has its own file. Potentially we could hide them under an `exceptions` or `errors` module if they get in the way of navigating the rest of the files.